doc/api/test.md

```diff
@@ -907,16 +907,19 @@ changes:
 * Returns: {TestsStream}
 
 ```mjs
-import { tap } from 'node:test/reporters';
+import { tap, run } from 'node:test/reporters';
 import process from 'node:process';
+import path from 'path';
 
 run({ files: [path.resolve('./tests/test.js')] })
   .compose(tap)
   .pipe(process.stdout);
 ```
 
 ```cjs
-const { tap } = require('node:test/reporters');
+const { tap, run } = require('node:test/reporters');
+const process = require('process');
```

Comment:
```suggestion
const { run } = require('node:test');
const { tap } = require('node:test/reporters');
```

Reply:
@MoLow This what @cjihrig is meant

Reply:
@OshriAsulin, yes @cjihrig also applies here, and in addition - Moshe suggests removing the `require` of `node:process`, which is already globally exposed